pdns/mbedtlscompat.hh

```diff
@@ -65,10 +69,12 @@ typedef md_type_t mbedtls_md_type_t;
 
 #define mbedtls_entropy_init entropy_init
 #define mbedtls_entropy_func entropy_func
+#define mbedtls_entropy_free entropy_free
 
-#define mbedtls_ctr_drbg_init
+#define mbedtls_ctr_drbg_init ctr_drbg_init
 #define mbedtls_ctr_drbg_seed ctr_drbg_init
```

Comment:
You are right, I have been fooled by the include/mbedtls/compat-1.3.h file shipped with mbedtls sources, which contains:
```
#define ctr_drbg_init mbedtls_ctr_drbg_init
```

I will revert that, thank you!